pandas/core/frame.py

```diff
@@ -391,7 +392,12 @@ def _constructor_expanddim(self):
     # ----------------------------------------------------------------------
     # Constructors
 
-    def __init__(self, data=None, index=None, columns=None, dtype=None, copy=False):
+    def __init__(self,
+                 data = None,
```

Comment:
Skipping type for `data` for now, as it's causing weird problems. Will be done in followups.
@WillAyd should I add a comment here to indicate annotation of data has been intentionally skipped for now?